app/src/main/res/layout/activity_main.xml

```diff
@@ -18,8 +18,16 @@
             android:background="?attr/colorPrimary"
             app:popupTheme="@style/AppTheme.PopupOverlay" />
 
+
     </android.support.design.widget.AppBarLayout>
 
+    <android.widget.ImageView
```

Comment:
literally no idea what this view is doing here, it is not used in the code expect of the mainactivity where it is assigned once for nothing. Therefore remove all changes from this file